src/node/node_state.h

```diff
@@ -1750,10 +1719,10 @@ namespace ccf
       return parse_create_response(response.value());
     }
 
-    bool create_and_send_request()
+    bool create_and_send_request(bool recovery = false)
```

Comment:
```suggestion
    bool create_and_send_init_request(bool recovery = false)
```

Or "genesis" if we like the name, otherwise it sounds much more generic than it really is.